roles/customized_ami/README.md

```diff
@@ -11,13 +11,28 @@ AWS credentials with valid permission.
 Role Variables
 --------------
 
-* **custom_ami_operation** - Operation to perform. Valid values are 'create', 'delete'.
-* **custom_ami_name** - Name of the AMI to create. **Required**
-* **custom_ami_packages** - List of packages to install.
-* **custom_ami_recreate_if_exists** - Whether to recreate the AMI if it already exists, default value is False.
-* **source_ami_filters** - A dict of filters to apply to find the source AMI id. See [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html) for possible filters.
-* **source_ami_image_id** - Image id of the AMI to be used as source. When not specified, we will use **source_ami_filters** to determine source AMI image id.
-* **source_ami_user_name** - User name to connect to EC2 instance used to create custom AMI. Default value is **ec2-user**.
+* **custom_ami_operation** (str) - Operation to perform. Valid values are 'create', 'delete'. Default: 'create'.
```

Comment:
Is any of these options required? I guess it will be valuable if you'll specify that.

Reply:
Typically, if a default value is specified, it is assumed that the argument is optional. I see that is what is followed in other roles' docs.  Please let me know if you think otherwise.

Reply:
That’s true. However, is custom_ami_packages required? I would assume that it isn’t, but It’s the only one not defaulting anything.

Reply:
No it is not required. No default value is needed. This is my understanding based on [this](https://github.com/redhat-cop/cloud.aws_ops/blob/main/roles/customized_ami/tasks/create_ami.yaml#L3).